modules/microvm/virtualization/microvm/appvm.nix

```diff
@@ -95,7 +95,10 @@ let
                 runWaypipe
                 pkgs.tpm2-tools
                 pkgs.opensc
-              ];
+              ]
+              ++ (lib.optional (
+                  config.ghaf.profiles.debug.enable )
+                    pkgs.pulseaudio );
```

Comment:
What's this used for? Setting hardware.pulseaudio.enable to true automatically adds pulseaudio to systemPackages.

Reply:
It could be leftover from when using pipewire instead of pulseaudio.
I can check that thanks.